### ISSUE TITLE
ebay-lightbox-dialog: Add subtitle

### ISSUE THREAD

**Issue by soniamacias** [eBay/evo-web#39]
### Description

Update Lightbox to support the "subtitle" line available in the Figma component. 

Not having it requires a workaround to get the correct a11y read order and introduces some unintended spacing. See [slack thread](https://ebay-eng.slack.com/archives/C03SVLC6669/p1742497002676739) for reference.

### Engineering pre-requisites

- [ ] Supports dark mode.
- [ ] Supports responsive design.
- [ ] Tokens have been provided (or already exist).

**Comment by dlacefield** [comment]
@dlacefield To follow up on specs
